versions.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 0.13.1"
+  required_version = ">= 1.0"
```

Comment:
Please make the same changes in all `versions.tf` (including examples).

Reply:
Yes, sure, thx

Reply:
Fixed.

Reply:
I will update VPC module version in examples to fix TF validate command.